src/20230831_AaveV3_Pol_SupplyCapLSTs/SupplyCapLSTs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "SupplyCapLSTs"
+author: "Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
+discussions: "https://governance.aave.com/t/arfc-supply-cap-increase-lsts-on-polygon-v3/14696"
+---
+
+## Simple Summary
+This proposal seeks to increase the supply cap for the wstETH and stMatic tokens on the AAVE Polygon V3 markets.
+
+This proposal is compatible with the Direct-to-AIP framework.
+
+## Motivation
+On the AAVE Polygon V3 market, the utilization of the supply cap reached 95% for stMatic and 100% for the wstETH, preventing the deposit of more assets for the latter. By augmenting those caps we would be helping the ecosystems that have been built around those products and increasing the user experience of the AAVE users, resulting in higher TVL and protocol revenue through the borrowing of WMatic and WETH.
+
+The total supplies of those assets increased recently, which allows the adjustement of those caps while staying below [75%](https://snapshot.org/#/aave.eth/proposal/0xf9261916c696ce2d793af41b7fe556896ed1ff7a8330b7d0489d5567ebefe3ba) of the total supplies.
```

Comment:
Is the supply of wstETH on Polygon relevant since it's bridged? (right?)

Reply:
it's not but that's a convo for risk teams lol.